contracts/BaseStrategy.sol

```diff
@@ -272,39 +272,34 @@ abstract contract BaseStrategy {
     bool public emergencyExit;
 
     // modifiers
-    modifier onlyAuthorized() {
-        require(msg.sender == strategist || msg.sender == governance(), "!authorized");
-        _;
+    function _onlyAuthorized() internal {
+        require(msg.sender == strategist || msg.sender == governance());
     }
 
-    modifier onlyEmergencyAuthorized() {
-        require(
-            msg.sender == strategist || msg.sender == governance() || msg.sender == vault.guardian() || msg.sender == vault.management(),
-            "!authorized"
-        );
-        _;
+    function _onlyEmergencyAuthorized() internal {
+        require(msg.sender == strategist || msg.sender == governance() || msg.sender == vault.guardian() || msg.sender == vault.management());
     }
 
-    modifier onlyStrategist() {
-        require(msg.sender == strategist, "!strategist");
-        _;
+    function _onlyStrategist() internal {
+        require(msg.sender == strategist);
     }
 
-    modifier onlyGovernance() {
-        require(msg.sender == governance(), "!authorized");
-        _;
+    function _onlyGovernance() internal {
+        require(msg.sender == governance());
     }
 
-    modifier onlyKeepers() {
+    function _onlyKeepers() internal {
         require(
             msg.sender == keeper ||
                 msg.sender == strategist ||
                 msg.sender == governance() ||
                 msg.sender == vault.guardian() ||
-                msg.sender == vault.management(),
-            "!authorized"
+                msg.sender == vault.management()
         );
-        _;
+    }
+
+    function onlyManagement() internal {
```

Comment:
if you are getting rid of all modifiers, kill this one as well.

Reply:
That's a request from https://github.com/yearn/yearn-vaults/issues/413 but can only be added by strategist.